_posts/2023-05-12-sprint-attendees.md

```diff
@@ -0,0 +1,39 @@
+---
+author: DjangoCon Europe Organisers
+category: General
+date: 2023-05-12
+layout: post
+published: true
+title: "Sprints: What are they & why should I attend?"
+---
+
+(In the ancient days of FLOSS…) Development sprints are most common around Free/Libre Open Source Software, or FLOSS projects. Most of these projects (like Django and Python) are maintained by a community of volunteers. These volunteers usually work together while spread around the world. Conferences presented rare occasions for these developers to work together in person. And thus, sprints were born! In the context of a given conference or event, you’ll often hear them referred to as “the sprints” or just “sprints”.
+
+Over time, these core developers realized sprints also provided them an opportunity to help find new potential contributors. And that’s where YOU come in.
+
+## Why Attend a Sprint at DJEU 2023?
+Sprints give you an opportunity to work shoulder-to-shoulder with experts in the community. In addition, there are practical and personal benefits to attending the sprints.
+
+From a practical perspective, if your organization makes heavy use of Django or a related package, getting involved in its maintenance will give you a better understanding of that package, as well as it’s strengths & limitations. On the personal side, most people won’t usually work on a project that’s as widely used as Django or some of the other related packages. Getting involved in the maintenance of one of these packages can expose you to engineering practices you like wouldn’t see otherwise. This can help you become a better engineer while allowing you to contribute to the community that benefits you and your company.
+
+## What should you know if you want to sprint?
+Git and Github basics
+Most FLOSS projects these days use Git for version control (the software used for tracking changes to software). This makes it easier for groups of people to work on the same software, together. If you’re new to Git, I recommend the first three chapters of Pro Git, which is freely available to read online or to download. Those chapters will get you started and give you about 80% of the Git you’ll use most often. Chapters five and six, on Distributed Git and Github respectively, are also very helpful if you’re new to working with a group on software.
```

Comment:
Could we put a link to the Pro Git book? This one, right? 
https://git-scm.com/book/en/v2